sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Cast.scala

```diff
@@ -1028,7 +1029,7 @@ case class Cast(child: Expression, dataType: DataType, timeZoneId: Option[String
       from: DataType,
       ctx: CodegenContext): CastFunction = from match {
     case StringType =>
-      val zid = ctx.addReferenceObj("zoneId", zoneId, "java.time.ZoneId")
+      val zid = ctx.addReferenceObj("zoneId", zoneId, classOf[ZoneId].getName)
```

Comment:
super nit: From the consistency view, I am curious that only this statement is not wrapped by `JavaCode.global`.  
Of course, it is not a big deal.

Reply:
Thanks for the comment, @kiszk -san!

I actually debated a bit with myself before sending out the PR. I was leaning towards the minimal change here, but since `zid` is used immedately in a `code` block, you're right that for style consistency we should wrap this in a `JavaCode.global`.

Actually I'm curious why this line of code didn't trigger the need to use `inline` when interpolating the string...